manifests/backends/bind.pp

```diff
@@ -19,33 +19,33 @@
   powerdns::config { 'bind-config':
     ensure  => present,
     setting => 'bind-config',
-    value   => "${powerdns::params::authoritative_configdir}/named.conf",
+    value   => "${powerdns::authoritative_configdir}/named.conf",
     type    => 'authoritative',
-    require => Package[$powerdns::params::authoritative_package],
+    require => Package[$powerdns::authoritative_package_name],
   }
 
-  file { "${powerdns::params::authoritative_configdir}/named.conf":
+  file { "${powerdns::authoritative_configdir}/named.conf":
     ensure  => file,
     mode    => '0644',
     owner   => 'root',
-    group   => 'root',
-    require => Package[$powerdns::params::authoritative_package],
+    group   => 0,
```

Comment:
Does changing this have any benefits on some operating systems? I haven't seen many systems with more users in group 0.

Reply:
There's no group 'root' on FreeBSD. 0 will be 'root' on Linux and 'wheel' on FreeBSD.

Reply:
Got it. It's been a really long time since I've worked with BSD :)